uni/lib/controller/fetchers/restaurant_fetcher.dart

```diff
@@ -0,0 +1,89 @@
+import 'package:http/http.dart';
+import 'package:tuple/tuple.dart';
+import 'package:uni/model/entities/restaurant.dart';
+import 'package:uni/model/entities/session.dart';
+
+import 'package:uni/controller/networking/network_router.dart';
+import 'package:uni/controller/parsers/parser_restaurants.dart';
+
+/// Class for fetching the menu
+class RestaurantFetcher {
+  String spreadSheetUrl = 'https://docs.google.com/spreadsheets/d/'
+      '1TJauM0HwIf2RauQU2GmhdZZ1ZicFLMHuBkxWwVOw3Q4';
+  String jsonEndpoint = '/gviz/tq?tqx=out:json';
+
+  // Range containing the meals table
+  // Format: Date(dd/mm/yyyy), Meal("Almoço", "Jantar), Dish("Sopa", "Carne",
+  //         "Peixe", "Dieta", "Vegetariano", "Salada"), Description(String)
+  String range = "A:D";
```

Comment:
make this variable more explicit, like

```suggestion
  String excel_column_range = "A:D";
```

So there is no need for the first comment line